README.md

```diff
@@ -151,18 +151,33 @@ In [firebase console](https://console.firebase.google.com/), you can get `google
 
 ## Usage
 
+### Receiving Notifications
 ```javascript
 import FCM from 'react-native-fcm';
 
 class App extends Component {
   componentDidMount() {
     FCM.requestPermissions(); // for iOS
+    
+    FCM.getInitFCMData().then((initialData)=>{
+      /* initialData will not be null if :
+      1- Android, app in background when notification received
+      2- Android, app killed when notification received
+      3- iOS, app killed when notification received
+      */
+    })
+    
     FCM.getFCMToken().then(token => {
       console.log(token)
       // store fcm token in your server
     });
     this.notificationUnsubscribe = FCM.on('notification', (notif) => {
-      // there are two parts of notif. notif.notification contains the notification payload, notif.data contains data payload
+      /*
+      This function will be called if :
+        1- Android and app is in the foreground ( notif['opened_from_tray'] will be set to false )
```

Comment:
you can and I think it is recommended to use `action` property in notification. and this property should be included only when you click the banner. Can you check if you can use that (combine with AppState) to differentiate the scenario?